tests/unit/ai/test_cursor_provider.py

```
@@ -162,6 +162,21 @@ def test_complete_one_shot_skips_resume(provider):
         cmd = mock_run.call_args.args[0]
         assert_that(cmd).does_not_contain("--resume")
 
+    def test_timeout_floor_is_six_hundred_seconds(self, provider):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `test_timeout_floor_is_six_hundred_seconds`

Add return type annotation: `None`

(ANN202)